integration-tests/tests/src/tests/sync/sync-session.ts

```diff
@@ -49,6 +50,7 @@ const PersonForSyncSchema: Realm.ObjectSchema = {
   },
 };
 
+// TODO: Fix this.
```

Comment:
This is unrelated to this PR, I can create a follow-up issue but waitForConnectionState seems to be a complete noop right now, may mean some tests either don't need it or are meaningless.

Reply:
https://github.com/realm/realm-js/issues/6759